Jellyfin.Api/Controllers/NotificationsController.cs

```diff
@@ -86,26 +87,19 @@ public IEnumerable<NameIdPair> GetNotificationServices()
         /// <summary>
         /// Sends a notification to all admins.
         /// </summary>
-        /// <param name="url">The URL of the notification.</param>
-        /// <param name="level">The level of the notification.</param>
-        /// <param name="name">The name of the notification.</param>
-        /// <param name="description">The description of the notification.</param>
+        /// <param name="notificationDto">The notification request.</param>
         /// <response code="204">Notification sent.</response>
         /// <returns>A <cref see="NoContentResult"/>.</returns>
         [HttpPost("Admin")]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
-        public ActionResult CreateAdminNotification(
-            [FromQuery] string? url,
-            [FromQuery] NotificationLevel? level,
-            [FromQuery] string name = "",
-            [FromQuery] string description = "")
+        public ActionResult CreateAdminNotification([FromBody, Required] AdminNotificationDto notificationDto)
```

Comment:
Is this correct? The ServiceStack attributes have it as `query`. I know that ServiceStack largely didn't care either way, but still...

Reply:
I think this is the main place that these endpoints are used, so might as well keep compat
https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserProxy.cs

Also, in the 10.6 controller it actually says `// This endpoint really just exists as post of a real with sickbeard` (whatever that was supposed to mean)

https://github.com/jellyfin/jellyfin/blob/release-10.6.z/Emby.Notifications/Api/NotificationsService.cs#L156-L173